ParisBNF/abb/BNFabb126.xml

```diff
@@ -0,0 +1,309 @@
+              <citedRange unit="number">126</citedRange>
+           </bibl>
+
+           <bibl>
+              <ptr target="bm:ContiRossini1914Notice"/>
+              <citedRange unit="page">185-186</citedRange>
+              <citedRange unit="number">174</citedRange>
+           </bibl>
+
+        </listBibl>
+      </source>
+      </recordHist>
+      </adminInfo>
+      </additional>
+
+               <msPart xml:id="p1">
+               <msIdentifier></msIdentifier>
+<msContents>
+
+
+<msItem xml:id="p1_i1">
+<locus from="1r" to="50v"/>
+<title ref="LIT1501GadlaS"></title>
+<textLang mainLang="gez"/>
+
+<colophon xml:id="coloph1">
+<locus target="50v"/>
+ለዘ፡ አጽሃፈኒ። አቡየ፡ አባኖብ፡ ጸሐፍክዋ፡ ለዛቲ፡ መጽሐፍ፡ አነ፡ አብርሃም። ዝክሩኒ፡ ወኢትርሥዑኒ፡ በበምንባቡ፡ ለአባሲኖዳ፡ ኦደቂቀ፡ <supplied reason="lost">ዳ</supplied>ጋ። ለዓለመ፡ ዓለም፡ ኤሜ<supplied reason="lost">ን።</supplied>
+<note>According to the colophone the commissioner was ʾabbā Nob and the scribe name is ʾAbrǝhām.</note>
+</colophon>
+</msItem>
+
+<msItem xml:id="p1_i2">
+<locus from="51r" to="73r"/>
+<title ref="LIT6537Besoy"></title>
+<note>Indication for reading on <locus target="#51r">f. 51r</locus>: አመ፡ ፰፡ ለኀምሌ፡</note>
+<textLang mainLang="gez"/>
+
+
+<colophon xml:id="coloph2">
+<locus target="73r"/>
+ጸሐፍክዋ፡ ለዛቲ፡ መጽሐፍ፡ አብርሃም፡ ኀጥዕ፡ ወዐባሲ፡ ዘገደፍኩ፡ በቃልየ፡ አርትዑ፡ በቃላቲክሙ። በእንተ፡ ፍቅረ፡ <sic>ተንሳዕ፡</sic> ክርስቶስ። አጽሐፈኒ፡ አቡየ፡ አባኖብ። ኅቡረ፡ ይምኅረነ፡ እግዚአብሔር፡
+በመንግሥተ፡ <sic>ሠማያት፡</sic> ለዓለመ፡ ዓለም፡ አሜን።
+<note>According to the colophone the commissioner was ʾabbā Nob and the scribe name is ʾAbrǝhām.</note>
+</colophon>
+</msItem>
+
+</msContents>
+
+<physDesc>
+<objectDesc>
+
+<layoutDesc>
+  <layout columns="2" writtenLines="32"/>
+</layoutDesc>
+
+</objectDesc>
+
+<handDesc>
+<handNote xml:id="h1" script="Ethiopic">
+<date notBefore="1350" notAfter="1500"/>
+</handNote>
+</handDesc>
+
+</physDesc>
+
+</msPart>
+
+<msPart xml:id="p2">
+<msIdentifier></msIdentifier>
+<msContents>
+
+<msItem xml:id="p2_i1">
+<locus from="74r" to="130v"/>
+<title ref="LIT4074GadlaMA"></title>
+<textLang mainLang="gez"/>
+
+<colophon xml:id="coloph3">
+<locus target="136v"/>
+ጸሎቱ፡ ወበረከቱ፡ የሃሉ፡ ምስለ፡ ጸሐፊሁ፡ ዘርአ፡ አብርሃም፡ ወምስለ፡ አጽሐፊሁ፡ ገብረ፡ መርዓዊ፡ ወምስለ፡ ኵሎሙ፡ ደቂቀ፡ ዳጋ፡ ለዓለመ፡ ዓለም፡ አሜን።
+<note>According to the colophone the scribe's name is Zarʾa ʾAbrǝhām and the commissioner was Gabra Marʿāwi.</note>
+</colophon>
+</msItem>
+
+</msContents>
+
+<physDesc>
+
+<objectDesc>
+  <layoutDesc>
+    <layout columns="2" writtenLines="25"/>
+  </layoutDesc>
+</objectDesc>
+
+<handDesc>
+<handNote xml:id="h2" script="Ethiopic">
+<date notBefore="1350" notAfter="1500"/>
+</handNote>
+</handDesc>
+
+<additions>
+<list>
+
+<item xml:id="a1">
+<locus target="130v"/>
+<desc></desc>
+<q xml:lang="gez">ለዘ፡ ጸሐፎ፡ ወለዘ፡ አጽሐፎ፡ ወለዘ፡ አንበቢ፡ ወለዘ፡ ተርጉሞ፡ ወለዘ፡ ሰምዓ፡ ቃላቲሁ፡ ሕቡረ፡ ይምሐሮው፡ እግዚአብሔር፡ በመንግሥተ፡ ሰማያት። ወፈድፋደሰ፡
+  ለእሉ፡ አኃዊነ፡ <add place="above">ገብረ፡ መርዓዊ፡</add> ፍሬ፡ ሰንበት፡ ዘም<supplied reason="omitted">፡</supplied> ወጽሕቁ፡ ለአጽሐፎቱ፡ ገድለ፡ ገብረ፡ መንፈስ፡ ቅዱስ። ይጽሐፍ፡
```

Comment:
IDs for the scribe and commissioner can be created. I would encode the material in a different way, because it is all just an extension of the explicit (may be a colophon from "ዘንተ፡ ገድለ፡ መንፈስ፡ ቅዱስ፡", a narrative colophon of the "old type")